apps/web/lib/utils/csrf.ts

```
@@ -0,0 +1,47 @@
+// If you see type errors for 'process' or 'crypto', ensure you have @types/node installed
+// npm i --save-dev @types/node
+import { cookies } from "next/headers";
+import { randomBytes } from "crypto";
```

Biome (1.9.4)

[error] A Node.js builtin module should be imported with the node: protocol.

Using the node: protocol is more explicit and signals that the imported module belongs to Node.js.
Unsafe fix: Add the node: protocol.


(lint/style/useNodejsImportProtocol)